pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -5531,18 +5531,19 @@
       "StableVersions": [
         "4.6.0",
         "4.7.0",
-        "4.7.1"
+        "4.7.2"
       ],
       "BaselineVersion": "4.7.0",
       "InboxOn": {
         "netcoreapp3.0": "4.0.0.0",
-        "netcoreapp3.1": "4.0.1.1",
+        "netcoreapp3.1": "4.0.1.2",
```

Comment:
This change is wrong, you actually need to revert this one to 4.0.1.0. I believe I tagged you on the issue explaining why, but basically we should never update inboxVersion during servicing

Reply:
yes i thought i reverted this one

Reply:
@joperezr  i tried that actually but i get this error while building the package
```
C:\Users\anagniho\.nuget\packages\microsoft.dotnet.build.tasks.packaging\1.0.0-beta.20113.5\build\Packaging.targets(1161,5): error : File System.Text.Json, version 4.0.1.2 was included framework package Microsoft.Private.CoreFx.NETCoreApp/4.7.0-dev.20176.1 but that version is not considered inbox in package index C:\git\corefx\pkg\Microsoft.Private.PackageBaseline\packageIndex.json.  Please add it with appropriate InboxOn entry for netcoreapp3.1 or suppress this message with PermitInbox suppression
```

whts the correct way to fix this ? cc @ericstj

Reply:
I believe you need to do this: https://github.com/dotnet/corefx/commit/57f547c6d0483e5f0f91f9583a8f4e89257983d9#diff-ca5f77b0975c879008d7369ffd167981R66 but I'll let others confirm

Reply:
Looks like with your latest changes you just reverted it back to 4.0.1.1, we actually need it to go back 4.0.1.0 instead. Can you fix that please?

Reply:
I don't understand, why did we downgrade this from `4.0.1.1`?

Reply:
The package reference to System.Text.Json will get dropped because the 4.0.1.1 will be considered inbox which wont be true for all versions of the sdk.

Reply:
In other words 4.0.1.0 was inbox in `netcoreapp3.1`.  4.0.1.1 was not inbox in `netcoreapp3.1`.  It's only inbox in a `netcoreapp3.1` shared frameork that had servicing updates.  There is no way to express *servicing updates* (nor a desire to) in the TFM `netcoreapp3.1` so the package index must not represent servicing updates being inbox in TFMs.

Reply:
Also worth noting, is that @Anipik change here is actually fixing something that didn't get introduced in this PR. A previous servicing PR updated this from 4.0.1.0 to 4.0.1.1 which was incorrect, we detected that later on and that is why I requested @Anipik to fix it here since he was already touching this. Just wanted to give you a bit more background to help understand.